spec/fixtures/plugin_modules/my_plug/tasks/resolve_reference.json

```diff
@@ -1,7 +1,8 @@
 {
-    "parameters": {
-        "optional_config": {
-            "type": "Optional[String]"
-        }
+  "description": "My custom plugin",
```

Comment:
Is there an integration test attached to these?

Reply:
Ah, no, I just assumed the fixtures were for an integration test but I see where they're used https://github.com/puppetlabs/bolt/pull/2872/files#diff-22978a4806ecd2ae6fd64f939c4ee442249a4fe838a399c43d459536f078ca1bR53